Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInSignUp.kt

```diff
@@ -152,12 +153,16 @@ fun Email(
             },
         textStyle = MaterialTheme.typography.body2,
         isError = emailState.showErrors(),
-        keyboardOptions = KeyboardOptions.Default.copy(imeAction = imeAction),
+        keyboardOptions = KeyboardOptions.Default.copy(
+            imeAction = imeAction,
+            keyboardType = KeyboardType.Email
+        ),
         keyboardActions = KeyboardActions(
-            onDone = {
+            onAny = {
```

Comment:
I don't think we'd want to use onAny here because that means onImeAction will also be called when next or previous (which should move focus) are triggered. Is there a case where onDone isn't enough?